src/App.tsx

```diff
@@ -1,14 +1,29 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from './app/hooks';
+import { actions as todosActions } from './features/todos';
+
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { TodoModal } from './components/TodoModal';
+
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const dispatch = useDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getTodos()
```

Comment:
There is no error handling for the `getTodos` API call. If the API request fails, the user will not be informed and the loading spinner will continue indefinitely.

Reply:
Fix